irc/server.py

```diff
@@ -436,14 +436,14 @@ def handle_dump(self, params):
                 print("     ", client.nick, client)
 
     def handle_ison(self, params):
-        response = '303: '
+        response = ':%s 303 %s :' % (self.server.servername, self.client_ident().nick)
```

Comment:
I know I have seen throughout the project a mixed use of {x} style formatting and the old % style. Do we have an absolute preference?

Reply:
`nickOnline.append(nick)/iso ` -> `nickOnline.append(nick)`?
What the preference is, that I don't know but I think I read in some PEP style guide/recommendation that the format {x} should be used..?

Reply:
No clue how that got in there... fixed :-) . And yeah as far as I know the only thing about using % is that it is slightly outdated (but I haven't found it to be deprecated)